guides/v2.1/cloud/reference/docker-config.md

```diff
@@ -80,27 +82,70 @@ The `ece-tools` package provides a `docker:build` command to generate the Docker
     ```bash
     docker-compose run build cloud-build
     ```
-    
+
     * Deploy Magento in the Docker container:
-    
+
     ```bash
     docker-compose run deploy cloud-deploy
     ```
 
+## Launch a Cloud Docker environment, `ece-tools` v2002.0.12
+
+Use `ece-tools` 2002.0.12 to generate the Docker compose configuration and deploy {{site.data.var.ece}} to a writeable file system in the Docker container. To deploy to a read-only file system, upgrade to `ece-tools` v2002.0.13 or later.
+
+1.  Download a template from the [Magento Cloud repository](https://github.com/magento/magento-cloud){:target="\_blank"}.
+1.  Add your credentials to `auth.json` file.
+1.  Update the template dependencies.
+
+    ```bash
+    composer install
+    ```
+
+1.  In your local environment, start the Docker configuration generator.
+
+    ```bash
+    vendor/bin/ece-tools docker:build
+    ```
+
+1.  Copy the configuration files.
+
+    ```bash
+	cp docker/config.env.dist docker/config.env
+    ```
+
+    ```bash
+	cp docker/global.env.dist docker/global.env
+    ```
+
+1.  Build files to containers and run in the background.
+
+    ```bash
+    docker-compose up -d --build
+    ```
+
+1.  Install Magento. This step may take some time to complete.
+
+    ```bash
+    docker-compose run cli magento-installer
+    ```
+
+1.  Open the `http://localhost:8080` URL in a browser to access your local Magento Cloud template.
+
+
 1. Access your local Magento Cloud template by opening one of the following secure URLs in a browser:
- 
+
     -  [`http://localhost:8080`](http://localhost:8080){:target="\_blank"}
     -  [`https://localhost`](https://localhost){:target="\_blank"}
 
-#### To stop and remove the Cloud Docker environment:
+## Stop and remove the Cloud Docker environment
```

Comment:
I'm okay with this not being a procedure title, but I think that the levels and order need more thought. 

Configure Docker (page title)
|_ ## Launch Docker Configuration
    |_ #### To launch Docker with ece 13
    |_ #### To launch Docker with ece 12
|_ ## Stop and Remove Docker configuration
|_ ## Integration testing

You get the idea...

Also, it is better to keep headings of the same level consistent...are we using gerunds? Launching, Stopping, Testing...they should remain consistent.